sources/launcher/Stride.Launcher/ViewModels/LauncherViewModel.cs

```diff
@@ -514,32 +503,27 @@ public async Task CheckForFirstInstall()
                         var versionToInstall = StrideVersions.First(x => x.CanBeDownloaded);
                         await versionToInstall.Download(true);
                     }
-                    if (!VsixPackage.IsLatestVersionInstalled && VisualStudioVersions.AvailableVisualStudioInstances.Any())
+
```

Comment:
I actually think that the VSIX install dialogs shouldn't be presented unless the user has elected to install some version of Stride.  It's a little bit of a corner case but makes a little more sense in my head that way.

Thoughts?

Reply:
I might have misunderstood you but this is already the case? When downloading the first version of the engine, the launcher will ask after downloading it whether the user would like to install the VS plugin

Reply:
It does, yes, but it will also ask if you _decline_ to install any version of Stride.   As I said, it's a bit of a corner case but feels weird to ask the user to install the extensions after they declined to install a first version of Stride.

Reply:
Sure, that makes sense, we can go ahead with your idea.